src/PythonMigrationViewExtension/IronPythonInfoDialog.xaml

```diff
@@ -51,12 +50,57 @@
 
             <TextBlock Name="DescriptionText"
                        Text="{x:Static w:Resources.IronPythonDialogDescription}"
-                       FontSize="12"
+                       FontSize="13"
                        Foreground="#bbbbbb"
                        HorizontalAlignment="Stretch"
-                       TextWrapping="Wrap">
+                       TextWrapping="Wrap"
+                       Margin="0,0,0,10">
             </TextBlock>
 
+            <!-- CustomNodes that have IronPython2 Dependency -->
+            <Expander x:Name="MainExpander" Header="{x:Static w:Resources.CustomNodesPythonDependencyHeader}" Foreground="#bbbbbb"
+                      FontSize="15" Margin="-2,0,0,0" Visibility="Hidden" ScrollViewer.VerticalScrollBarVisibility="Visible">
+                <StackPanel Height="Auto">
+                    <Expander x:Name="ExpanderForPackagedDependencies" Header="Packaged" Foreground="#bbbbbb" FontSize="15" Margin="20,5,0,0" Visibility="Collapsed">
+                        <ListBox x:Name="PackagedCustomNodesContainingPython" FontSize="12" Background="Black" Height="Auto" MaxHeight="100"
+                                 Margin="25,5,0,0" ScrollViewer.VerticalScrollBarVisibility="Visible" SelectionMode="Extended">
+                            <ListBox.CommandBindings>
+                                <CommandBinding Command="ApplicationCommands.Copy" CanExecute="CanExecuteCtrlCCopyCommand" Executed="ExecuteCtrlCCopyCommand" />
+                            </ListBox.CommandBindings>
+                            <ListBox.ItemTemplate>
+                                <DataTemplate>
+                                    <Grid Margin="0,2" ToolTip="{x:Static w:Resources.PackagedCustomNodesTooltip}">
+                                        <Grid.ColumnDefinitions>
+                                            <ColumnDefinition Width="100" />
+                                            <ColumnDefinition Width="*" />
+                                        </Grid.ColumnDefinitions>
+                                        <TextBlock Foreground="#bbbbbb" Text="{Binding PackageName}" />
+                                        <TextBlock Grid.Column="1" Foreground="#bbbbbb" Text="{Binding Name}" />
```

Comment:
can we use colors only from the dynamo resource dictionary and if we need new colors add them to the dictionary with clear names?

Reply:
Using the colors from resource dictionary now.